Error: Reverse for 'nika_app_view' not found. 'nika_app_view' is not a valid view function or pattern name. - Reverse for 'nika_app_view' not found. 'nika_app_view' is not a valid view function or pattern name.
Traceback (most recent call last):
  File "C:\inetpub\wwwroot\sitefolder\myproject\.\myapp\views.py", line 356, in home_view
    return render(request, 'home.html', context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\loader_tags.py", line 208, in render
    return template.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 177, in render
    return self._render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "c:\inetpub\wwwroot\sitefolder\django_iis_env\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'nika_app_view' not found. 'nika_app_view' is not a valid view function or pattern name.
Internal Server Error: /home/
